nixos-modules/valheim.nix

```diff
@@ -147,8 +190,19 @@ in {
                 cp $cfg $out/$(stripHash $cfg)
               done
             '';
+          createListFile = name: list: ''
+              echo "// List of Steam IDs for ${name} ONE per line" > ${stateDir}/.config/unity3d/IronGate/Valheim/${name}
+              ${lib.concatMapStrings (id: "echo '${id}' >> ${stateDir}/.config/unity3d/IronGate/Valheim/${name}\n") list}
```

Comment:
We should not use a map function for side-effects.  I think you can invoke `echo` just once with the result of `lib.strings.concatStringsSep "\n" list`.

Reply:
Resolved in [904cd1b](https://github.com/aidalgol/valheim-server-flake/pull/13/commits/904cd1b165e4768b5b8caf62a39a32427cb50f1a)